nextroute/schema/input.go

```diff
@@ -7,129 +7,217 @@ import (
 
 // Input is the default input schema for nextroute.
 type Input struct {
-	Options        any              `json:"options,omitempty"`
-	CustomData     any              `json:"custom_data,omitempty"`
-	Defaults       *Defaults        `json:"defaults,omitempty"`
-	StopGroups     *[][]string      `json:"stop_groups,omitempty"`
-	DurationMatrix *[][]float64     `json:"duration_matrix,omitempty"`
-	DistanceMatrix *[][]float64     `json:"distance_matrix,omitempty"`
+	// Options arbitrary options.
+	Options any `json:"options,omitempty"`
+	// CustomData arbitrary custom data.
+	CustomData any `json:"custom_data,omitempty"`
+	// Defaults default properties for vehicles and stops.
```

Comment:
Another sentence like this would be helpful to me:
"Default values are applied to stops and vehicles that do not set them explicitly."